CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ We encourage pull requests and other contributions from the community. Before su
 
 ### Prerequisites
 
-The .NET Standard target requires only the .NET Core 2.1 SDK or higher, while the iOS and Android targets require the corresponding Xamarin SDKs.
+The .NET 7 target requires only the .NET Core 2.1 SDK or higher, while the iOS, Android, MacCatalys, and Windows targets require the corresponding MAUI SDKs.
```

Comment:
I have concerns, because this doesn't make sense to me.

Reply:
Fixed.